flag_engine/features/schemas.py

```diff
@@ -47,30 +47,20 @@ class FeatureStateSchema(Schema):
         allow_none=True,
         required=False,
     )
-    segment_id = fields.Method(
-        serialize="serialize_segment_id",
-        deserialize="deserialize_segment_id",
-        allow_none=True,
-        required=False,
-    )
     multivariate_feature_state_values = ListOrDjangoRelatedManagerField(
         fields.Nested(MultivariateFeatureStateValueSchema)
     )
 
+    class Meta:
+        unknown = EXCLUDE
```

Comment:
Is this for the documents that already have `segment_id` ?

Reply:
Yep, exactly. We might need to add it to all the schemas tbh as we're likely to have another scenarios like this where we deprecate attributes.